src/Controller/Admin/OIDCCrudController.php

```diff
@@ -43,17 +44,24 @@ public function configureFields(string $pageName): iterable
             $domains = $this->siteRepository->getPrimaryDomains();
             yield ChoiceField::new('domain')
                 ->setChoices(array_combine($domains, $domains))
-            ;
+                ->setLabel('Site');
         } else {
             yield TextField::new('domain')
-                ->setTemplatePath('admin/domain.html.twig');
+                ->setTemplatePath('admin/domain.html.twig')
+                ->setLabel('Site');
         }
 
+        yield ChoiceField::new('type')
+            ->hideOnForm()
+            ->setChoices(ServerTypeType::CHOICES)
+            ->renderExpanded()->setColumns(8)
+            ->setTemplatePath('EasyAdminBundle/Fields/server_type.html.twig');
+
         yield UrlField::new('onePasswordUrl')
             ->setLabel(new TranslatableMessage('1Password url'));
         yield UrlField::new('usageDocumentationUrl')->hideOnIndex()
             ->setHelp(new TranslatableMessage('Tell where to find documentation on how OpenID Connect is used on the site and
 how to configure the use.'));
-        yield DateTimeField::new('expirationTime');
+        yield DateField::new('expirationTime')->setFormat('yyyy-MM-dd')->setLabel('Expiration Date');
```

Comment:
No fan of the hardcode date format here. Should it be moved into configuration.

Reply:
This is for display/UI only. On the entity and db level it's a full `DateTime` field.